### PR TITLE
feat: added admin full name TISNEW-5104

### PR DESCRIPTION
**Pull request by jayanta2018**
added admin full name as atrributes in the AWS Cognito

### CODE REVIEW

File: src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java [hunk 1]
```diff
@@ -60,9 +62,34 @@
...
-    return listUsersResult.getUsers().stream()
-        .map(mapper::toDto)
-        .collect(Collectors.toList());
+    return buildAdminDtoList(userTypeList);
+
+  }
+
+  private List<AdminDto> buildAdminDtoList(List<UserType> userTypeList) {
+    List<AdminDto> adminDtoList = new ArrayList<>();
+    for (UserType userType : userTypeList) {
```
Comment (uzairbaig) [inline src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java:73]: Please use streaming instead of this loop.

**Review by uzairbaig — CHANGES_REQUESTED** [review]

File: src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java (hunk 1 above)
Comment (jayanta2018) [reply src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java]: @uzairbaig I have added streaming

File: src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java [hunk 2]
```diff
@@ -60,9 +62,34 @@
...
+
+  }
+
+  private List<AdminDto> buildAdminDtoList(List<UserType> userTypeList) {
+    List<AdminDto> adminDtoList = new ArrayList<>();
+    userTypeList.stream().forEach(userType -> {
+      AdminDto adminDto = new AdminDto();
+      adminDto.setUsername(userType.getUsername());
+      adminDto.setFullName(getUserFullName(userType.getAttributes()));
+      adminDtoList.add(adminDto);
```
Comment (uzairbaig) [inline src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java:77]: Please use .Collect() method instead of manually adding into list.

**Review by uzairbaig — CHANGES_REQUESTED** [review]

File: src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java (hunk 2 above)
Comment (jayanta2018) [reply src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java]: @uzairbaig I have upadated the code. Thanks

**Review by uzairbaig — APPROVED** [review]

**Pull request merged**